src/fides/api/ops/api/v1/endpoints/registration_endpoints.py

```diff
@@ -0,0 +1,81 @@
+import logging
+
+from fastapi import Depends
+from sqlalchemy.orm import Session
+from starlette.exceptions import HTTPException
+
+from fides.api.ops.api import deps
+from fides.api.ops.api.v1 import urn_registry as urls
+from fides.api.ops.models.registration import UserRegistration
+from fides.api.ops.schemas import registration as schemas
+from fides.api.ops.util.api_router import APIRouter
+
+router = APIRouter(
+    tags=["Registration"],
+    prefix=urls.V1_URL_PREFIX,
+)
+
+logger = logging.getLogger(__name__)
+
+
+@router.get(
+    urls.REGISTRATION,
+    status_code=200,
+    response_model=schemas.GetRegistrationStatusResponse,
+)
+def get_registration_status(
+    *,
+    db: Session = Depends(deps.get_db),
+) -> schemas.GetRegistrationStatusResponse:
+    """
+    Return the registration status of this Fides deployment.
+    """
+    registrations = UserRegistration.all(db=db)
+    if registrations:
+        registration = registrations[0]
+        opt_in = registration.opt_in
+    else:
+        opt_in = False
+
+    return schemas.GetRegistrationStatusResponse(opt_in=opt_in)
+
+
+@router.put(
+    urls.REGISTRATION,
+    status_code=200,
+    response_model=schemas.Registration,
+)
+def get_registration_status(
```

Comment:
At some point, this needs to hit fideslog. This is going to be hard to code out before ethyca/fideslog#87 is merged and the new methods are available in the SDK. 

You also have a naming collision here with L26. I remember my first copy/paste job... 🤔